### DISCUSSION TITLE
Credit card transfers - using rules

### DISCUSSION THREAD

**Discussion by Dinth** [firefly-iii/firefly-iii#7770] [General]
Hi. Sorry for the another credit card question,
I've got a several credit cards, which I'm trying to pay back monthly - with a payment usually split for interest+minimal payment via Direct Debit (automatic) and the balance paid manually. 
I'm using those cards to pay for the majority of purchases and importing the data from all the accounts through Data Importer
The problem is that I cannot use the account mapping during, which was previously recommended here for handling credit cards - The credit card payments are being imported as Unknown destination account, and it's not possible to map them correctly. 
So normally I'm ending with two transactions for each of the card payment. Additionally, one of my cards is in a foreign currency so im using a third party service for payments (sending money to Wise, and Wise is topping up my credit card - but im sometimes also using Wise for other transfers)

How can i deal with such situation without using mappings? Right now, I'm using Firefly III to categorize my spendings, but all the overall values like "Spending" or account balances are messed up because of my credit card payments

**Comment by JC5** [comment]
Mmmm. I'm not sure if you can, especially not using rules. They can only handle so much complexity.

**Comment by Dinth** [comment]
But besides the foreign currency card, this is a fairly standard usage of credit cards, therefore im just wondering how other people are working around that
